### ISSUE TITLE
Tests failing for Python 3.8 and Python 3.9

### ISSUE THREAD

**Issue by shayandoust** [intake/intake#550]
Hello,

It seems like pytest is failing for the Python versions mentioned in the title. Hence, this is blocking an upload, which rectifies a RC bug.

[Here] is the output of the tests. Please let me know if there is anything I can try to rectify these tests failing, or if you require any further information.

I do see some errors within the log like
```
   def info(self, path, **kwargs):
        path = self._strip_protocol(path)
>       out = os.stat(path, follow_symlinks=False)
E       FileNotFoundError: [Errno 2] No such file or directory: '/<<PKGBUILDDIR>>/.pybuild/cpython3_3.8_intake/build/intake/source/tests//calvert_uk_filter.tar.gz'

/usr/lib/python3/dist-packages/fsspec/implementations/local.py:60: FileNotFoundError
```
however, nothing has been touched and I'm not too sure why these files are missing.

Thanks,
Shayan

[Here]: https://paste.debian.net/1172603/

**Comment by martindurant** [comment]
Is it possible to find out the set of package versions that are used in the test, and the instal method? Intake's CI does test against python 38, so that won't be the issue.

**Comment by martindurant** [comment]
Did you have time to delve deeper into this?

**Comment by shayandoust** [comment]
Hello martindurant,

Apologies for the lengthy absence. It seems like that issue was to do with what files were to be included into the autopkgtest, so that issue above now stands resolved. Now there persists a final issue, which I think started happening only for Python 3.9 for the latest release tarball of intake.

Here is the bug report opened: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=982699
If you would like a full build log from my system, please let me know. Hopefully the issue is trivial.

Hopefully this is the final issue we can get resolved as we are entering the next stage of Debian freezes.

**Comment by martindurant** [comment]
I believe that is a result of a new version of pandas, and different conventions for data type matching therein. It's probably already passing on master. 

Would our releasing a new version solve things for you?

**Comment by shayandoust** [comment]
Yes please. Once you've released a new version I will test it and let you know of any issues.

**Comment by martindurant** [comment]
The new version is on pypi and conda-forge
